host/tests/tvSettings_L3_Tests/tvSettings_test32_ComponentLuma.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+#** *****************************************************************************
+# *
+# * If not stated otherwise in this file or this component's LICENSE file the
+# * following copyright and licenses apply:
+# *
+# * Copyright 2024 RDK Management
+# *
+# * Licensed under the Apache License, Version 2.0 (the "License");
+# * you may not use this file except in compliance with the License.
+# * you may obtain a copy of the License at
+# *
+# * http://www.apache.org/licenses/LICENSE-2.0
+# *
+# * Unless required by applicable law or agreed to in writing, software
+# * distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+# *
+#* ******************************************************************************
+
+import os
+import sys
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+sys.path.append(os.path.join(dir_path, "../"))
+
+from tvSettings_L3_Tests.tvSettingsHelperClass import tvSettingsHelperClass
+
+class tvSettings_test32_ComponentLuma(tvSettingsHelperClass):
+
+    # Predefined luma values ranging from 1 to 30
+    lumaValues = [1, 5, 10, 15, 30]
+    rackDevice = "dut"
```

Comment:
remove this, this variable is already part of helper function

Reply:
removed